integrations/jenkins/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,35 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: jenkinsJob
+    selector:
+      query:  "true"
+    port:
+      entity:
+        mappings:
+          identifier: 'if .url | contains("://") then (.url | split("://")[1] | sub("^.*?/"; "")) else .url end | sub("%20"; "-"; "g") | sub("/"; "-"; "g") | .[:-1]'
```

Comment:
I prefer sticking with ids over the url
the url might change between jenkins versions